pep_sphinx_extensions/pep_theme/templates/redirect.html

```diff
@@ -0,0 +1,16 @@
+{# Redirect template #}
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="refresh" content="0; url={{ pathto(pep_file) }}">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>PEP Redirector | peps.python.org</title>
+    <link rel="shortcut icon" href="{{ pathto('_static/py.png', resource=True) }}"/>
+    <link rel="canonical" href="{{ pathto(pep_file) }}" />
+    <meta name="description" content="Python Enhancement Proposals (PEPs)"/>
+</head>
+<body>
+    <p><a href="{{ pathto(pep_file) }}">PEP {{ pep_num }}</a></p>
+</body>
```

Comment:
```suggestion
```

If we don't want the page to render, then we shouldn't include a `<body>`, which will flash onscreen (as you noticed) as well as cause a non-trivial delay to load, parsing and rendering.